from pymongo.errors import ConnectionFailure

MONGO_NETWORK_ERRORS = ConnectionFailure
